### PR TITLE
update iot-app-kit to 3.4

### PR DESCRIPTION
**Pull request by MO-Elmu**
**What this PR does / why we need it**:
Update iot-app-kit to 3.4 and ready to release a new plugin version.

**Which issue(s) this PR fixes**:
This PR upgrade to a new AppKit version. This AppKit version introduces new features and fixes

Fixes #
- fix visibility of data overlays on parent objects
- add entity binding support
- fix raycast issues
- add support for selection events on data overlays

**Special notes for your reviewer**:
The main purpose of this PR is a new plugin release that contain the above AppKit features and fixes. We already made a plugin from the branch and thoroughly tested it last week.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -92,11 +92,11 @@
...
-    "@iot-app-kit/core": "3.3.0",
-    "@iot-app-kit/react-components": "3.3.0",
-    "@iot-app-kit/scene-composer": "3.3.0",
-    "@iot-app-kit/source-iottwinmaker": "3.3.0",
-    "aws-iot-twinmaker-grafana-utils": "file:libs/aws-iot-twinmaker-grafana-utils-1.0.0.tgz",
+    "@iot-app-kit/core": "3.4.0",
+    "@iot-app-kit/react-components": "3.4.0",
+    "@iot-app-kit/scene-composer": "3.4.0",
+    "@iot-app-kit/source-iottwinmaker": "3.4.0",
+    "aws-iot-twinmaker-grafana-utils": "file:libs/aws-iot-twinmaker-grafana-utils-2.0.0.tgz",
```
Comment (hwandersman) [inline package.json:99]: I don't think we want this to be upgraded to 2.0

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v1.6.3
+-fix visibility of data overlays on parent objects
+-add entity binding support
```
Comment (hwandersman) [inline CHANGELOG.md:5]: I don't know what these changes are referring to

Comment (MO-Elmu) [reply CHANGELOG.md:5]: Weston is not here today, However these are AppKit changes.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v1.6.3
+-fix visibility of data overlays on parent objects
```
Comment (mukeshsahay) [inline CHANGELOG.md:4]: Please fix the formatting as per previous releases

File: package.json (hunk 1 above)
Comment (mukeshsahay) [reply package.json:99]: That should be ok..

File: CHANGELOG.md (hunk 3 above)
Comment (mukeshsahay) [reply CHANGELOG.md:4]: This is published here and looks like it is not working - https://grafana.com/grafana/plugins/grafana-iot-twinmaker-app/?tab=changelog

**Review by mukeshsahay — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (MO-Elmu) [reply CHANGELOG.md:5]: This refer to the entity binding support introduced in 3DKG previously binding was supported only through widgets like tags ..etc no you can do entity binding without the needs for widgets like tags ..etc.

File: CHANGELOG.md (hunk 3 above)
Comment (MO-Elmu) [reply CHANGELOG.md]: Fixed

Comment (MO-Elmu) [reply CHANGELOG.md]: it was missing the space after the Dashes

**Review by hwandersman — APPROVED** [review]

**Review by mukeshsahay — APPROVED** [review]

**Pull request merged**